config/examples/ttl.yml

```diff
@@ -0,0 +1,15 @@
+ttl: &default
+  account_deletion:
+    account_inactivity: 365
+    account_suspension: 90
+    contract_unpaid_time: 183
+
+development:
+  <<: *default
+
+test:
+  <<: *default
+
+production:
```

Comment:
So it's disabled in production?

Reply:
By default yes, but we don't use this for SaaS, we will use https://github.com/3scale/deploy/pull/537 so it will be enabled 😄

Reply:
It is the actual config for OpenShift since we're missing `openshift/system/config/examples/ttl.yml`.

Reply:
Ok, I will write a specific file for OpenShift then because for on-prem it should be disabled by default.

Reply:
And same for Docker 😄

Reply:
Gui said:
> f we want them disabled and in `config/examples/ttl.yml` you didn’t include `&default` for `production:`, then it is disabled already

So that's it 😄 No need to add anything else 💃